### ISSUE TITLE
Generating sample after training (CIFAR-10)

### ISSUE THREAD

**Issue by chang0517** [y0ast/Glow-PyTorch#27]
I had difficulty writing code using TensorFlow. thank you for providing with an easier code using PyTorch. However, I have two issues while using the code, so I am leaving this comment."

1. I completed the training using CIFAR-10, but even though I used the optimizer 'adamax' as written in the Readme, the NLL loss is coming out as 3.48. Was this loss not meant for the CIFAR-10 dataset?
2. After completing the training, I generated images, but only black images appear and no images are generated. Is the publicly available ipynb file supposed to be working?
